src/method_signatures.md

```diff
@@ -54,33 +54,33 @@ A method that takes a borrowed parameter cannot assign the parameter to a member
 
 To indicate a parameter is borrowed, in the method signature, place a “&” in front of the type.
 
-As an example, in Java you might define a method 'isSorted’ like this 
+As an example, in Java you might define a method `isSorted()` like this 
 ```java
 boolean isSorted(List<Integer> values);
 ```
-that returns a boolean indicating if a list is sorted. In Rust you would add an ‘&’ in front of the type in the method signature to indicate that it will not retain any references to the list or it's contents when the method returns.
+that returns a boolean indicating if a list is sorted. In Rust you would add an ‘&’ in front of the type in the method signature to indicate that it will not retain any references to the list nor its contents when the method returns.
 ```rust ,skt-default
 # trait Example {
 fn is_sorted(values: & Vec<i32>) -> bool;
 # }
 ```
-It is helpful to think of 'borrowed’ as being part of the type. IE “The method takes a borrowed list.” 
+It is helpful to think of ‘borrowed’ as a part of the type(i.e. “The method takes a borrowed list.”) 
 
-This declaration provides a strong and useful guarantee to the callers of a method. But it would be worthless if in a newer version, the method just deleted the ‘&’ from it's signature and removed the guarantee. To prevent this, and to make the guarantee explicit in the caller's code, when a method is being invoked; When a method that borrows a parameter is invoked, the caller puts an ‘&’ in front of the variable name being passed in. For example 
+This declaration provides a strong and useful guarantee to the callers of a method. But it would be worthless if in a newer version, the method just deleted the ‘&’ from its signature and removed the guarantee. To prevent this and to make the guarantee explicit in the caller's code, when a method that borrows a parameter is invoked, the caller puts an ‘&’ in front of the variable name being passed in. For example 
 ```rust ,skt-main
 # fn is_sorted(values: & Vec<i32>) -> bool { true }
 # let values = vec![1, 2, 3];
 if (is_sorted(&values)) {
     //...
 }
 ```
-This is only needed if the variable being passed isn't itself borrowed. (Otherwise it would be redundant, because it can only pass it to methods which borrow it.)
+This is only needed if the variable being passed isn't self-borrowed. (Otherwise it would be redundant, because it can only pass it to methods which borrow it.)
```

Comment:
I find that a bit confusing. Does: "This is only needed if the variable being passed isn't borrowed itself." make sense to you?
```suggestion
This is only needed if the variable being passed isn't borrowed itself. (Otherwise it would be redundant, because it can only pass it to methods which borrow it.)
```

Reply:
"self-borrowed" is semantically the same as "borrowed by itself"(I guess this is what you meant), but "the variable being passed isn't borrowed ifself" is syntactically confusing(possibly incorrect). 

Besides, I think the issue that is more critical is that there's no such concept of "self-borrowed" in Rust, I think.

Reply:
> Besides, I think the issue that is more critical is that there's no such concept of "self-borrowed"

Right, but the phrase is odd enough that saying sounds like it's implying there is a concept of "self-borrowed" which I'd like to avoid.

Reply:
What about: "This is only needed if the variable being passed isn't already borrowed."